src/rgw/rgw_admin.cc

```diff
@@ -2010,6 +2014,17 @@ static void get_md_sync_status(list<string>& status)
     push_ss(ss, status) << "metadata is caught up with master";
   } else {
     push_ss(ss, status) << "metadata is behind on " << total_behind << " shards";
+    
+    string shards_behind_str;
+    shards_behind_str.append("[");
+    set<string>::iterator it = shards_behind_set.begin();
+    for ( ; it != shards_behind_set.end(); it++) {
+      shards_behind_str.append(*it + ", ");
+    }
+    shards_behind_str.erase(shards_behind_str.size()-2);
+    shards_behind_str.append("]");
```

Comment:
it looks like this could be simplified by using a `std::set<int>`, and taking advantage of its `operator<<` defined in `include/types.h`:
```c++
std::set<int> shards = {0,1,2,3};
std::cout << '[' << shards << ']'; // prints [0,1,2,3]
```